Build failure with profiling enabled

In file included from /2TB/usr/src/github.com/1div0/vvdec/source/Lib/vvdec/../CommonLib/CommonDef.h:583,
                 from /2TB/usr/src/github.com/1div0/vvdec/source/Lib/DecoderLib/DecLibParser.h:45,
                 from /2TB/usr/src/github.com/1div0/vvdec/source/Lib/DecoderLib/DecLib.h:49,
                 from /2TB/usr/src/github.com/1div0/vvdec/source/Lib/DecoderLib/DecLibRecon.cpp:47:
/2TB/usr/src/github.com/1div0/vvdec/source/Lib/DecoderLib/DecLibRecon.cpp: In lambda function:
/2TB/usr/src/github.com/1div0/vvdec/source/Lib/DecoderLib/DecLibRecon.cpp:686:27: error: ‘picture’ was not declared in this scope; did you mean ‘Picture’?
  686 |       __itt_frame_end_v3( picture->m_itt_decLibInst, nullptr );
      |                           ^~~~~~~
/2TB/usr/src/github.com/intel/ittapi/include/ittnotify.h:247:39: note: in definition of macro ‘ITTNOTIFY_VOID_D1’
  247 | #define ITTNOTIFY_VOID_D1(n,d,x)     (d == NULL) ? (void)0 : (!(d)->flags) ? (void)0 : (!ITTNOTIFY_NAME(n)) ? (void)0 : ITTNOTIFY_NAME(n)(d,x)
      |                                       ^
/2TB/usr/src/github.com/1div0/vvdec/source/Lib/DecoderLib/DecLibRecon.cpp:686:7: note: in expansion of macro ‘__itt_frame_end_v3’
  686 |       __itt_frame_end_v3( picture->m_itt_decLibInst, nullptr );
      |       ^~~~~~~~~~~~~~~~~~
make[2]: *** [source/Lib/vvdec/CMakeFiles/vvdec.dir/build.make:636: source/Lib/vvdec/CMakeFiles/vvdec.dir/__/DecoderLib/DecLibRecon.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:198: source/Lib/vvdec/CMakeFiles/vvdec.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

Attached CMake cache file for reference.
[CMakeCache.txt](https://github.com/fraunhoferhhi/vvdec/files/11575022/CMakeCache.txt)

--- a/source/Lib/DecoderLib/DecLibRecon.cpp
+++ b/source/Lib/DecoderLib/DecLibRecon.cpp
@@ -683,7 +683,7 @@ void DecLibRecon::decompressPicture( Picture* pcPic )
 
 #ifdef TRACE_ENABLE_ITT
       // mark end of frame
-      __itt_frame_end_v3( picture->m_itt_decLibInst, nullptr );
+      __itt_frame_end_v3( param->pic->m_itt_decLibInst, nullptr );
 #endif
       param->pic->stopProcessingTimer();
 

--- a/source/Lib/vvdec/CMakeLists.txt
+++ b/source/Lib/vvdec/CMakeLists.txt
@@ -116,6 +116,8 @@ if( VVDEC_ENABLE_X86_SIMD )
   endif()
 
   add_library( ${LIB_NAME}_x86_simd OBJECT ${SSE41_SRC_FILES} ${SSE42_SRC_FILES} ${AVX_SRC_FILES} ${AVX2_SRC_FILES} )
+  target_link_libraries( ${LIB_NAME}_x86_simd ${INTEL_ITT_LINK_TARGET} )
+
   # disble LTO for the files compiled with special architecture flags
   set_target_properties( ${LIB_NAME}_x86_simd PROPERTIES
                                               INTERPROCEDURAL_OPTIMIZATION                OFF

